src/components/Exchange/PositionSeller.js

```diff
@@ -114,6 +153,18 @@ export default function PositionSeller(props) {
   const prevIsVisible = usePrevious(isVisible);
   const positionRouterAddress = getContract(chainId, "PositionRouter");
   const nativeTokenSymbol = getConstant(chainId, "nativeTokenSymbol");
+  const toTokens = getTokens(chainId);
+
+  const [savedRecieveTokenAddress, setSavedRecieveTokenAddress] = useLocalStorageByChainId(
+    chainId,
+    `${CLOSE_POSITION_RECEIVE_TOKEN_KEY}-${position?.indexToken?.symbol}-${position.isLong ? 'long' : 'short'}`
+  );
+
+  const [swapToken, setSwapToken] = useState(() => (
```

Comment:
i'm thinking what if address stored in local storage is invalid? let's consider local storage as another source of user data – not trust it

Reply:
If address is invalid in this line swapToken will be initiated as undefined, it's ok

```
const [swapToken, setSwapToken] = useState(() => (
    savedRecieveTokenAddress 
      ? toTokens.find(token => token.address === savedRecieveTokenAddress)
      : undefined
  ));
 ```